Tools/ros2/ardupilot_sitl/src/ardupilot_sitl/launch.py

```diff
@@ -282,21 +282,21 @@ def generate_action(context: LaunchContext, *args, **kwargs) -> ExecuteProcess:
 
         # Retrieve launch arguments.
         master = LaunchConfiguration("master").perform(context)
-        # out = LaunchConfiguration("out").perform(context)
+        out = LaunchConfiguration("out").perform(context)
         sitl = LaunchConfiguration("sitl").perform(context)
 
         # Display launch arguments.
         print(f"command:          {command}")
         print(f"master:           {master}")
         print(f"sitl:             {sitl}")
+        print(f"out:              {out}")
 
         # Create action.
         mavproxy_process = ExecuteProcess(
             cmd=[
                 [
                     f"{command} ",
-                    "--out ",
-                    "127.0.0.1:14550 ",
+                    f"--out {out} ",
```

Comment:
> And it probably need default value to "127.0.0.1:14550 " in case nothing is passed to retain the current behavior

That's already handled by the `generate_launch_arguments` method which supplies a default for a single `--out` param.

> should use a list as argument so you can add as much as you want .

Yep. Could revisit the current launch methods in a follow up PR and apply this consistently. Will prob need to handle both list and single value as fallback to support existing usage.

Reply:
I agree with the list of --out, in general you may want to pass more than one argument at a time. And yes, the `localhost:14550` is default, so you don't need to add it. One option could be to have localhost anyway and add an external --out argument in case it was different